docker_scripts/ubuntu-clang.dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG UBUNTU_VERSION=latest
+FROM ubuntu:${UBUNTU_VERSION}
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update -y -q && \
+    apt-get upgrade -y -q && \
+    apt-get install -y -q --no-install-recommends \
+        ca-certificates \
+        clang \
+        cmake \
+        git \
+        libeigen3-dev \
```

Comment:
in some sense, i would prefer to make this dock image independent of eigen and just fetch eigen as a step inside the CI file in pressio directly

Reply:
Done, using eigen `3.4.0` release from GitLab.